Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -34,6 +48,9 @@ import BraintreeCore
         self.sepaDirectDebitAPI = sepaDirectDebitAPI
     }
 
+    @objc func applicationDidBecomeActive(notification: Notification) {
+        returnedToAppAfterPermissionAlert = isAuthenticationSessionStarted
```

Comment:
q: is this needed only for analytics?

Reply:
yes, for special cancel from the alert.

Reply:
Would tracking this be valuable? The only concern I have is we're adding application code only for analytics. We may have an opportunity to simplify tracking if we remove this event.

Reply:
Ah ok 👍 .